cartography/data/permission_relationships.yaml

```diff
@@ -0,0 +1,18 @@
+# map principals who can read items in a S3Bucket
```

Comment:
I know we talked about different config formats, but we could also consider writing this simply as a python file as described in this [blog](https://beepb00p.xyz/configs-suck.html) and shown in this [popular example project](https://github.com/getpelican/pelican/blob/master/pelican/settings.py).

Anyone else have any other thoughts?